registry/cloudflare-workers/serverless.yml

```diff
@@ -9,3 +9,39 @@ inputTypes:
         script_path:
                 type: string
                 required: true
+        account_id:
```

Comment:
Format YAML consistently with rest of serverless project (2 spaces for tabs)

Reply:
This was giving us quite the headache for some reason, 2 spaces were giving us an error that only fixed when switching back to the headache inducing 8 :(